uber/common.py

```diff
@@ -70,6 +70,7 @@
 from uber.amazon_ses import AmazonSES, EmailMessage  # TODO: replace this after boto adds Python 3 support
 from uber.config import c, Config, SecretConfig
 from uber.utils import *
+from uber.reports import *
```

Comment:
(Catching up on my old reviews.)

I would NOT do an ``from uber.reports import *`` here and instead *strongly* recommend just saying ``import uber.reports``.

Why?  Because part of the contract of the ``uber.reports`` module is that plugins might change some of its fields.  If a site section refers to ``printed_badge_report_type`` directly instead of referring to ``uber.reports.printed_badge_type``, then those two could literally be different classes.  That's not good, and I wouldn't expect future plugin authors to necessarily understand the difference.

We should therefore remove the possibility of referring to e.g. ``printed_badge_type`` directly by NOT doing a ``from uber.reports import *`` here, since that's just asking for trouble.